src/server/express/a2a_express_app.ts

```diff
@@ -32,6 +32,20 @@ export class A2AExpressApp {
         const router = express.Router();
         router.use(express.json(), ...(middlewares ?? []));
 
+        router.use((err, req, res, next) => {
+            // Check if it's the specific JSON parsing error
```

Comment:
If this is a known issue and there is some prior art, let's reference it in the comment here.